src/test/ui/consts/min_const_fn/min_const_fn_unsafe_feature_gate.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
```

Comment:
Entirely unrelated to this file...
But I think it might be good to replicate https://github.com/rust-lang/rust/blob/master/src/test/ui/consts/min_const_fn/min_const_fn_libstd_stability.rs for `const unsafe fn` just to be on the safe side.